### DISCUSSION TITLE
Is there a way to limit the height of the selected items box and show a vertical scrollbar?

### DISCUSSION THREAD

**Discussion by danielinteractive** [selectize/selectize.js#1692] [Q&A] [answered]
Apologies if this is a naive question, I am coming to selectize.js via R/Shiny so I am really a total newbie with all this.

Basically the question is: can we have a vertical scrollbar in the selected items in selectize.js? Currently we have the situation that the more items we select, they are all showing up in the box, and therefore the box becomes larger and larger - taking more and more vertical space. 

Is there a css or js option we could use to set the maximum height of the selected items box and if that height is exceeded a vertical scrollbar is used?

Example:
![image](https://user-images.githubusercontent.com/1071227/139227817-1e31f1d4-9ca3-475e-93dd-c01f5a04ba0c.png)

**Comment by danielinteractive** [comment, marked as answer]
OK I found the answer:
```css
.selectize-input {
  max-height: 102px;
  overflow-y: auto;
}
```

**Reply by duju211** [reply]
Hi! How does this look in your shiny app? It would be ineteresting for me to see, how one can set the option in shiny! Thx for your help!

**Reply by danielinteractive** [reply]
Hi @duju211 , please have a look at https://github.com/insightsengineering/teal.modules.hermes/blob/main/inst/css/custom.css where this is used.
